utils/docker/Dockerfile.centos.8

```diff
@@ -187,17 +189,17 @@ RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || \
       TARGET_TYPE=$DAOS_TARGET_TYPE --deps-only && \
     ([ "$DAOS_KEEP_BUILD" != "no" ] || /bin/rm -rf build *.gz); }
 
-# force an upgrade to get any newly built RPMs
+# force an upgrade to get any newly built RPMs, but only if CACHEBUST is set.
 USER root:root
 ARG CACHEBUST
-RUN dnf -y upgrade \
-        --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury && \
-    dnf clean all
+RUN [ -z "$CACHEBUST" ] || \
+    { dnf upgrade --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury && \
```

Comment:
not dnf -y ?

Reply:
The "dnf config-manager --save --setopt=assumeyes=True" above applies this in the config file so we can remove all the -y and --assumeyes options from there forward.